keyform/models.py

```diff
@@ -2,5 +2,60 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from django.conf import settings
+from django.utils.timezone import now
 
-# Create your models here.
+
+class Request(models.Model):
+
+    REQUEST_TYPES = (
+        ('bk', 'Broken Key'),
+        ('sk', 'Stolen Key'),
+        ('wk', 'Worker Key'),
+    )
+
+    PAYMENT_TYPES = (
+        ('ca', 'Cash'),
+        ('ch', 'Check'),
+    )
+
+    building = models.ForeignKey('Building')
```

Comment:
You may want to remove the quotes around Building to make it match all the other ForeignKey's